doc/source/dev/development_environment.rst

```diff
@@ -249,11 +266,20 @@ Now, you can run::
 
     $ spin gdb mytest.py
 
+In case you are using clang toolchain::
+
+    $ lldb python mytest.py
+
 And then in the debugger::
 
     (gdb) break array_empty_like
     (gdb) run
 
+lldb counterpart::
+
+    (gdb) breakpoint set --name array_empty_like
+    (gdb) run
```

Comment:
minor: I assume these prompts shouldn't be showing `(gdb)` for an `lldb` session?

Reply:
That's correct - fixed it.